.circleci/config.yml

```diff
@@ -21,6 +21,9 @@ jobs:
       - setup_remote_docker:
           docker_layer_caching: true
       - checkout
+      - run:
+          name: bake
```

Comment:
We also have a [`bake` step in our build workflows](https://github.com/greenpeace/planet4-international/blob/develop/.circleci/config.yml#L166), which does something quite different, could be confusing. Perhaps `generate-dockerfiles` instead?

Reply:
Or `bake-dockerfiles` would do if we still want some cake :grin:

Reply:
Yeah, good point. I can open a follow up PR, once I test the tag workflow works as expected.